### PR TITLE
fix: missing variable rename

### PR DESCRIPTION
**Pull request by zhouhanseng**

### CODE REVIEW

File: packages/maskbook/src/plugins/FileService/define.tsx [hunk 1]
```diff
@@ -44,6 +44,6 @@ export const FileServicePluginDefine: PluginConfig = {
             },
         ],
     ]),
-    pageInspector: FileServiceCompositionUI,
+    PageComponent: FileServiceCompositionUI,
```
Comment (zhouhanseng) [inline packages/maskbook/src/plugins/FileService/define.tsx:47]: https://github.com/DimensionDev/Maskbook/commit/2530b062b29708254c41ec4d458eb42bd28f6d1c#diff-2e0e90c36cac04931dfea7ddb0d1ecb0a25915cf093447cd91283aabc4658063L15

`pageInspector` has been renamed to `PageComponent`

File: tsconfig.json [hunk 2]
```diff
@@ -83,6 +83,6 @@
       ]
     },
     "transpileOnly": true,
-    "compiler": "ttypescript"
+    "compiler": "typescript"
```
Comment (guanbinrui) [inline tsconfig.json:86]: @Jack-Works

**Review by guanbinrui — DISMISSED** [review]

File: tsconfig.json (hunk 2 above)
Comment (Jack-Works) [reply tsconfig.json:86]: This is not a typo. ttypescript is a library.

**Review by Jack-Works — CHANGES_REQUESTED** [review]
Request change

**Review by septs — APPROVED** [review]

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**